web/src/dropdown_widget.ts

```diff
@@ -140,6 +144,7 @@ export class DropdownWidget {
         this.dropdown_input_visible_selector =
             options.dropdown_input_visible_selector ?? this.widget_selector;
         this.prefer_top_start_placement = options.prefer_top_start_placement ?? false;
+        this.dropdown_opened_by_keyboard_click = options.dropdown_opened_by_keyboard_click ?? false;
```

Comment:
Since this variable is used after the dropdown is initialized, we don't need to allow caller to provide initial value here.